README.md

```diff
@@ -111,7 +111,7 @@ provider you like.
    const oauth2Client = createGitHubOAuth2Client();
 
    async function handleAccountPage(request: Request) {
-     const sessionId = await getSessionId(request);
+     const sessionId = getSessionId(request);
      const isSignedIn = sessionId !== undefined;
```

Comment:
nit: Is this variable name accurate?

Reply:
I think so. Can you think of a more accurate name?

Reply:
It looks like `sessionId !== undefined` now only means the client has session id cookie, and it's not necessarily valid. I think `const isSignedIn = accessToken !== null` is more accurate way to use this variable name

Reply:
Ah, yes, I see. Perhaps, we should name is `hasSessionIdCookie`. WDYT?

Reply:
`hasSessionIdCookie` sounds fine!